lib/lte_link_control/lte_lc.c

```diff
@@ -854,67 +589,45 @@ int lte_lc_connect_async(lte_lc_evt_handler_t handler)
 		return -EINVAL;
 	}
 
-	return w_lte_lc_connect(false);
+	return connect_lte(false);
 }
 
 int lte_lc_init_and_connect_async(lte_lc_evt_handler_t handler)
 {
 	int err;
 
-	err = w_lte_lc_init();
+	err = init_and_config();
 	if (err) {
 		return err;
 	}
 
 	return lte_lc_connect_async(handler);
 }
 
-int lte_lc_offline(void)
-{
-	if (at_cmd_write(offline, NULL, 0, NULL) != 0) {
-		return -EIO;
-	}
-
-	return 0;
-}
-
-int lte_lc_power_off(void)
-{
-	if (at_cmd_write(power_off, NULL, 0, NULL) != 0) {
-		return -EIO;
-	}
-
-	return 0;
-}
-
 int lte_lc_deinit(void)
 {
 	if (is_initialized) {
 		is_initialized = false;
 		at_notif_deregister_handler(NULL, at_handler);
-		return lte_lc_power_off();
+		return lte_lc_func_mode_set(LTE_LC_FUNC_MODE_POWER_OFF);
 	}
 
 	return 0;
 }
 
 int lte_lc_normal(void)
 {
-	int err;
-
-	err = enable_notifications();
-	if (err) {
-		LOG_ERR("Failed to enabled notifications, error: %d", err);
-		return err;
-	}
+	return lte_lc_func_mode_set(LTE_LC_FUNC_MODE_NORMAL);
```

Comment:
What about the call to `enable_notifications()` which was added to fix CIA-272?

Reply:
Good catch, rebase accident. Fixed now.